LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
The Apache license contains a patent grant, which we cannot give. Also, most gems are licensed under MIT. I liked it.